html-forms/index.html

```diff
@@ -10,31 +10,32 @@ <h1>Coding Form!</h1>
       <div>
         <input type="text" name="email" id="email">
       </div>
-      <label for="coding options">Why is coding cool?</label>
+      <label for="coding-options">Why is coding cool?</label>
       <div>
-        <select name="coding options" id="coding options">
+        <select name="coding-options" id="coding-options">
           <option value="select">Select an option</option>
           <option value="interesting">Because it's interesting</option>
           <option value="fun">Because it's fun</option>
           <option value="powerful">Because it's powerful</option>
-          </div>
         </select>
-        <p>Favorite Language (choose one)</p>
-        <div>
-          <input type="radio" name="language" id="language" value="HTML"><label for="html">HTML</label>
-          <input type="radio" name="language" id="language" value="CSS"><label for="label">CSS</label>
-          <input type="radio" name="language" id="language" value="JavaScript"><label for="JavaScript">JavaScript</label>
-        </div>
-        <p>Would you like to sign up for more information?</p>
-        <div>
-          <input type="checkbox" id="Sign me up" name="Sign me up" ><label for="Sign me up">Sign me up!</label>
-        </div>
-        <label for="comments">Comments</label>
-        <div>
-          <textarea name="comments" id="comments" cols="32" rows="8"></textarea>
-        </div>
-        <div>
-          <input type="submit" value="Submit">
-        </div>
-      </body>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <input type="radio" name="language" id="HTML" value="HTML"><label for="html">HTML</label>
+        <input type="radio" name="language" id="CSS" value="CSS"><label for="label">CSS</label>
```

Comment:
These two `label` elements have a `for` attribute that doesn't match the `id` attribute on the `input` they are paired with.